techniques/general/G145.html

```diff
@@ -239,52 +180,44 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
                           /255</li>
                      </ul>
                      <div class="note">
-				<p class="note-title marker">Note</p>
+				<p class="note-title marker">注記</p>
 				<div>
-                        <p>The "^" character is the exponentiation operator.</p>
+                        <p>"^" 記号は指数演算子である。</p>
                      </div>
 			</div>
                   </li>
                </ul>
                <div class="note">
-				<p class="note-title marker">Note</p>
+				<p class="note-title marker">注記</p>
 				<div>
-                  <p>For aliased letters, use the relative luminance value found two pixels in
-                    from the edge of the letter.</p>
+                  <p>エイリアス文字では文字の端から 2 ピクセルの部分の相対輝度の値を使用する。</p>
                </div>
 			</div>
             </li>
-            <li>Measure the relative luminance of the background pixels
-                  immediately next to the letter using same formula.</li>
+            <li>同じ公式を用いて、文字のすぐ隣の背景のピクセルの相対輝度を測る。</li>
             <li>
-               <p>Calculate the contrast ratio using the following
-                  formula.</p>
+               <p>次の公式を用いて、コントラスト比を算出する。</p>
                <ul>
                   <li>
-                     <p>(L1 + 0.05) / (L2 + 0.05), where</p>
+                     <p>(L1 + 0.05) / (L2 + 0.05) ここで、</p>
                      <ul>
-                        <li>L1 is the
-                          <a href="https://w3c.github.io/wcag/guidelines/22/#dfn-relative-luminance" target="terms">relative luminance</a>
-                          of the lighter of the foreground or background colors, and</li>
-                        <li>L2 is the
-                          <a href="https://w3c.github.io/wcag/guidelines/22/#dfn-relative-luminance" target="terms">relative luminance</a>
-                          of the darker of the foreground or background colors.</li>
+                        <li>L1 は前景又は背景色の明るい方の<a href="https://w3c.github.io/wcag/guidelines/22/#dfn-relative-luminance" target="terms">相対輝度</a>である。及び、</li>
```

Comment:
「相対輝度」のリンク先を、WCAG 2.2 (WAIC による日本語訳) の当該用語のパスに変更しました。

```suggestion
                        <li>L1 は前景又は背景色の明るい方の<a href="https://waic.jp/translations/WCAG22/#dfn-relative-luminance" target="terms">相対輝度</a>である。及び、</li>
```